packages/terra-brand-footer/tests/wdio/brand-footer-spec.js

```diff
@@ -18,24 +18,77 @@ describe('BrandFooter', () => {
     });
   });
 
-  describe('Hovered', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-brand-footer/brand-footer/hovered-brand-footer');
+  describe('Links Prop', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-brand-footer/brand-footer/links-prop-brand-footer'));
+
+    Terra.should.beAccessible({ viewports, rules });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-brand-footer-nav-background-color': 'pink',
+      '--terra-brand-footer-nav-color': 'yellow',
+      '--terra-brand-footer-link-color': 'green',
+      '--terra-brand-footer-footer-content-background-color': 'red',
+      '--terra-brand-footer-footer-content-color': 'black',
     });
+  });
 
+  describe('No Nav', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-brand-footer/brand-footer/no-nav-brand-footer'));
+
+    Terra.should.beAccessible({ viewports, rules });
     Terra.should.matchScreenshot({ viewports });
     Terra.should.themeEachCustomProperty({
-      '--terra-brand-footer-link-hover-color': 'brown',
+      '--terra-brand-footer-footer-content-background-color': 'red',
+      '--terra-brand-footer-footer-content-color': 'black',
     });
   });
 
-  describe('Active', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-brand-footer/brand-footer/active-brand-footer');
+  describe('Section Headers', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-brand-footer/brand-footer/section-header-brand-footer'));
+
+    Terra.should.beAccessible({ viewports, rules });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
```

Comment:
Do we need to add themeable tests for Section Headers or are these vars covered in the previous tests?